### DISCUSSION TITLE
Neovim development

### DISCUSSION THREAD

**Discussion by ryo33** [jsinger67/parol#161] [General]
I'd like to register parol-ls to the following repositories to help users that use Vim/Neovim for development.

- https://github.com/mason-org/mason-registry
- https://github.com/neovim/nvim-lspconfig

**Comment by jsinger67** [comment]
Hi @ryo33,
I highly appreciate that.
I know that neovim is in need of parol-ls support from other users but since I have no experience with neovim I can't provide efficient support here.

**Comment by jsinger67** [comment]
I know that @AumyF provided with PR #78 the support for stdio to enable other editors like Neovim and Helix.

**Comment by ryo33** [comment]
Thanks! With the stdio option, I confirmed `vim.lsp.start({name="parol-ls", cmd={"parol-ls", "--stdio"}, root_dir = "."})` works in my Neovim.

**Comment by jsinger67** [comment]
Hi @ryo33,
could you make any progress in this case?
Do you need any support?

**Reply by ryo33** [reply]
Thank you for reaching out. I haven't made any progress yet, but I believe it's not difficult. It's just that I've been putting things off.

**Reply by jsinger67** [reply]
That's completely fine. Don't worry 😊
